docsource/buildinitclient.rst

```diff
@@ -76,50 +76,64 @@ Build the client::
 
     BUILD SUCCESSFUL
     Total time: 3 seconds
-    
+
 The client jar is created in ``dist/client/WorkspaceClient.jar``.
 
 For simplicity, copy the required jars into a single directory::
 
     bareubuntu@bu:~/ws$ mkdir tryjavaclient
     bareubuntu@bu:~/ws$ cd tryjavaclient/
     bareubuntu@bu:~/ws/tryjavaclient$ cp ../workspace_deluxe/dist/client/WorkspaceClient.jar .
-    bareubuntu@bu:~/ws/tryjavaclient$ cp ../jars/lib/jars/jackson/jackson-*-2.2.3.jar .
-    bareubuntu@bu:~/ws/tryjavaclient$ cp ../jars/lib/jars/kbase/auth/kbase-auth-0.3.1.jar .
-    bareubuntu@bu:~/ws/tryjavaclient$ cp ../jars/lib/jars/kbase/common/kbase-common-0.0.10.jar .
+    bareubuntu@bu:~/ws/tryjavaclient$ cp ../jars/lib/jars/jackson/jackson-*-2.5.4.jar .
+    bareubuntu@bu:~/ws/tryjavaclient$ cp ../jars/lib/jars/kbase/auth/kbase-auth-0.4.4.jar .
+    bareubuntu@bu:~/ws/tryjavaclient$ cp ../jars/lib/jars/kbase/common/kbase-common-0.0.24.jar .
     bareubuntu@bu:~/ws/tryjavaclient$ ls
-    jackson-annotations-2.2.3.jar  kbase-auth-0.3.1.jar
-    jackson-core-2.2.3.jar         kbase-common-0.0.10.jar
-    jackson-databind-2.2.3.jar     WorkspaceClient.jar
+
+    jackson-annotations-2.5.4.jar  jackson-jaxrs-json-provider-2.5.4.jar      WorkspaceClient.jar
+    jackson-core-2.5.4.jar         jackson-module-jaxb-annotations-2.5.4.jar
+    jackson-databind-2.5.4.jar     kbase-auth-0.4.4.jar
+    jackson-jaxrs-base-2.5.4.jar   kbase-common-0.0.24.jar
 
 When creating an application using the WSS it's advisable to use a build tool
 like ``ant``, ``maven``, or ``gradle`` to organize the required jars.
 
 This simple program initializes and calls a method on the WSS client::
 
-    bareubuntu@bu:~/ws/tryjavaclient$ cat TryWorkspaceClient.java 
+    bareubuntu@bu:~/ws/tryjavaclient$ cat TryWorkspaceClient.java
 
 .. code-block:: java
 
     import java.net.URL;
-
+    import us.kbase.auth.AuthConfig;
     import us.kbase.workspace.WorkspaceClient;
+    import us.kbase.auth.ConfigurableAuthService;
+    import us.kbase.auth.AuthToken;
 
     public class TryWorkspaceClient {
-	
+
         public static void main(String[] args) throws Exception {
+            String authUrlInsecure = "false";
+        String authUrl = "https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login/";
+
+            ConfigurableAuthService authService = new ConfigurableAuthService(
+                    new AuthConfig().withKBaseAuthServerURL(new URL(authUrl))
+                    .withAllowInsecureURLs("true".equals(authUrlInsecure)));
```

Comment:
Just stick **true** in instead of the string stuff IMO

Reply:
Or just take out the insecure urls part altogether since you're going against ci. Probably better not to show it and avoid encouraging bad habits